CMakeLists.txt

```diff
@@ -31,6 +36,19 @@ endif()
 
 include(${BLT_SOURCE_DIR}/SetupBLT.cmake)
 
+option(WITH_PYTHON_BINDINGS "Build/Install Python bindings for Adiak" FALSE)
```

Comment:
Should be renamed `ENABLE_PYTHON_BINDINGS` to be consistent with the BLT naming scheme

Reply:
[0e9ce20](https://github.com/LLNL/Adiak/pull/38/commits/0e9ce2086c2f39f70df9fa803bce9a3f373ff657) I will note this is `WITH_PYTHON_BINDINGS` in Caliper, though. Should we change it there too?